### ISSUE TITLE
'_GeometryService' object has no attribute 'areasAndLengths'

### ISSUE THREAD

**Issue by bixb0012** [Esri/arcgis-python-api#73]
Can someone explain why I am seeing this error?  Maybe I am misunderstanding the documentation, but I don't understand why this error is being raised.
```Python
>>> import arcgis
>>> from arcgis.gis import GIS
>>> from arcgis.geometry import Geometry
>>> 
>>> gis = GIS()
>>> 
>>> geom = Geometry({"rings":[[[0,0],[0,10],[10,10],[10,0],[0,0]],[[3,3],[7,3],[7,7],[3,7],[3,3]]],
...                  "spatialReference":{"wkid":26915}})
>>>
>>> arcgis.geometry.areas_and_lengths(geom, "", "", "", 26915)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Anaconda3\lib\site-packages\arcgis\geometry\functions.py", line 80, in areas_and_lengths
    return gis._tools.geometry.areasAndLengths(
AttributeError: '_GeometryService' object has no attribute 'areasAndLengths'
>>>
```

**Comment by rohitgeo** [comment]
@bixb0012 I can confirm that you've run into a bug. We'll get the fix out in the next update but till then a workaround will be to change
C:\Anaconda3\lib\site-packages\arcgis\geometry\functions.py", line 80 to the following:    
```python
    return gis._tools.geometry.areas_and_lengths(
```

**Comment by bixb0012** [comment]
@rohitgeo , thanks for confirming it is a bug and not me misunderstanding the documentation.  Also, thanks for the workaround.

**Comment by bixb0012** [comment]
Going to leave this issue open until the fix is actually deployed.

**Comment by rohitgeo** [comment]
The fix was included in v1.0.1 released in February. The api doc link is here:
http://esri.github.io/arcgis-python-api/apidoc/html/arcgis.geometry.html#areas-and-lengths
